plugins/woocommerce/includes/admin/helper/class-wc-helper-subscriptions-api.php

```diff
@@ -139,6 +139,8 @@ public static function get_subscriptions() {
 	 */
 	public static function refresh() {
 		WC_Helper::refresh_helper_subscriptions();
+		WC_Helper::get_subscriptions();
```

Comment:
Is this required? `self::get_subscriptions()` below seems to call `WC_Helper::get_subscription_list_data` which calls `get_subscriptions`.

Reply:
I think it's fine. `WC_Helper::get_subscriptions` will make the API request and cache it. When it comes to `WC_Helper::get_subscription_list_data()` call, the cache version will be used.